lua/user_example/init.lua

```diff
@@ -72,7 +71,6 @@ local config = {
 
   -- Diagnostics configuration (for vim.diagnostics.config({...})) when diagnostics are on
```

Comment:
We should probably edit this line to clarify that this is controlling when `vim.g.diagnostics_mode = 3` here